releasenotes/notes/asm-1-click-activation-with-RCM-6a2431949768b73b.yaml

```
@@ -1,6 +1,12 @@
 ---
+prelude: >
+  Application Security Management (ASM) has added support for preventing attacks by blocking malicious IPs using one click within Datadog.
+
+
+  .. note::
+    One click activation for ASM is currently in beta.
+
+
 features:
   - |
-    ASM: 1-click activation ability through Remote Configuration Management. 
-    Set ``DD_REMOTE_CONFIGURATION_ENABLED`` to true to enable 1-click activation for ASM. 
-    Note that this feature is currently in beta.
+    ASM: Added support for one click activation using Remote Configuration Management (RCM). Set ``DD_REMOTE_CONFIGURATION_ENABLED=true`` to enable this feature.
```

```suggestion
    ASM: add support for one click activation using Remote Configuration Management (RCM). Set ``DD_REMOTE_CONFIGURATION_ENABLED=true`` to enable this feature.
```

nit: we usually use present tense in our release notes